packages/client/src/components/Actions/components/Confirmations.js

```diff
@@ -0,0 +1,46 @@
+import Svg from "library/Svg";
+import { useContacts } from "hooks";
+import { getSafeContactName, shortenAddr, shortenName } from "utils";
+
+const Confirmations = ({ confirmations, safeData }) => {
+  const { address: safeAddress } = safeData;
+
+  const { contacts } = useContacts(safeAddress);
+
+  const getNumberOfConfirmations = () => {
+    const numberApproved = Object.keys(confirmations).filter(
+      (key) => confirmations[key] === "approved"
+    )?.length;
+
+    return `${numberApproved} out of ${Object.keys(confirmations).length}`;
+  };
+
+  const getConfirmationList = () => {
+    return Object.keys(confirmations).map((key) => {
+      const name = getSafeContactName(contacts, key);
+      return (
+        <div className="confirmation">
+          <Svg name="Status" className={`${confirmations[key]}`} />
+          {name?.length > 20 ? shortenName(name) : name}
```

Comment:
nit, suggest adding a couple more references for readability
```suggestion
         const displayAddress = name ? `·${shortenAddr(key)}` : key;
      const displayName = name?.length > 20 ? shortenName(name) : name;
      return (
        <div className="confirmation">
          <Svg name="Status" className={`${confirmations[key]}`} />
          {name ? `${displayName}${displayAddress}` : `${displayAddress}`}
        </div>
      );
```